src/frontend/src/app/asset-readings/chart-modal/chart-modal.component.ts

```diff
@@ -194,4 +210,9 @@ export class ChartModalComponent implements OnInit {
       datasets: ds
     };
   }
+
+  clearField(limitField, offsetField) {
+    limitField.inputValue = '';
```

Comment:
don;t we have binding of component html > ts fields binding with `this` ?

Reply:
that is not required. it's a local field in html tamplate and pass as a parameter in method